synfig-studio/src/gui/dialogs/dialog_guide.h

```diff
@@ -59,7 +60,11 @@ class Dialog_Guide : public Gtk::Dialog
 	WorkArea* current_work_area;
 
 	Gtk::SpinButton* angle_widget;
+	Gtk::SpinButton* x_widget;
+	Gtk::SpinButton* y_widget;
```

Comment:
Maybe you should use `Widget_Distance` :)

Your code does not allow user to user another distance measure unit other than the current one set on preferences.